src/cli/commands/object/get.js

```diff
@@ -21,32 +20,16 @@ module.exports = Command.extend({
       if (err) {
         throw err
       }
-      if (utils.isDaemonOn()) {
-        return ipfs.object.get(key, (err, obj) => {
-          if (err) {
-            log.error(err)
-            throw err
-          }
 
-          console.log(JSON.stringify(obj))
-        })
-      }
-
-      const mh = new Buffer(bs58.decode(key))
-      ipfs.object.get(mh, (err, obj) => {
+      ipfs.object.get(key, {enc: 'base58'}, (err, node) => {
```

Comment:
pretty sure @xicombd is going to get excited to see that now we don't need all the madness checking :)